moveit_ros/moveit_servo/src/servo_node.cpp

```diff
@@ -31,133 +31,220 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *******************************************************************************/
 
-/*      Title     : servo_node.cpp
- *      Project   : moveit_servo
- *      Created   : 12/31/2018
- *      Author    : Andy Zelenak
+/*      Title       : servo_node.cpp
+ *      Project     : moveit_servo
+ *      Created     : 01/07/2023
+ *      Author      : V Mohammed Ibrahim
+ *
  */
 
-#include <moveit_servo/servo_node.h>
+#include <moveit_servo/servo_node.hpp>
+#include <realtime_tools/thread_priority.hpp>
 
-static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_servo.servo_node");
+namespace
+{
+const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_servo.servo_node");
+// This value is used when configuring the main loop to use SCHED_FIFO scheduling
+// We use a slightly lower priority than the ros2_control default in order to reduce jitter
+// Reference: https://man7.org/linux/man-pages/man2/sched_setparam.2.html
+int const THREAD_PRIORITY = 40;
```

Comment:
This addition seems a bit hacky... could it at least be configurable?

Reply:
Updated this to be configurable parameter, will be useful to set priority without recompiling.